HealthChecks.Gcp.CloudFirestore/DependencyInjection/CloudFirestoreHealthCheckBuilderExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿using HealthChecks.Gcp.CloudFirestore;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class CloudFirestoreHealthCheckBuilderExtensions
+    {
+        const string NAME = "cloud firestore";
+
+        public static IHealthChecksBuilder AddCloudFirestore(this IHealthChecksBuilder builder, Action<CloudFirestoreOptions> setup, string name = default, HealthStatus? failureStatus = default, IEnumerable<string> tags = default)
```

Comment:
Add missing comments on this public surface, please check existing comments on others projects.